### PR TITLE
Maturation of getting started guide on web

### PR DESCRIPTION
**Pull request by simonlaustsen**
Did a few changes to the getting started guide related to some of the feedback received

### CODE REVIEW

File: src/shared/getting-started/congrats.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+## Summary
+
+You're at the end of your journey, and you've accomplished a lot! 🎉
+
+* You learned which prerequisites is needed to start building with MapsIndoors.
+* You loaded a interactive map with MapsIndoors locations and added a floor selector for navigating between floors.
+* You created a search experience to search for specific locations on the map.
+* You added functionality for getting directions from one location to another.
```
Comment (marksdk) [inline src/shared/getting-started/congrats.md:8]:
```suggestion
* You added functionality for getting directions from one Location to another.
```

File: src/shared/getting-started/congrats.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+* You learned how to enable different types of live data domains in your app.
```
Comment (marksdk) [inline src/shared/getting-started/congrats.md:9]:
```suggestion
* You learned how to enable different types of Live Data Domains in your app.
```

File: src/shared/getting-started/congrats.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
+This concludes the "Getting Started" tutorial. To get more inspiration on what to build next please visit our [showcase page](https://www.mapspeople.com/showcases) to see how other clients have used MapsIndoors.
```
Comment (marksdk) [inline src/shared/getting-started/congrats.md:11]:
```suggestion
This concludes the "Getting Started" tutorial. To get more inspiration on what to build next please visit our [showcase page](https://www.mapspeople.com/showcases) to see how other clients use MapsIndoors.
```

File: src/web/v4/getting-started/map.md [hunk 4]
```diff
@@ -103,6 +103,8 @@ After you added the script tag into `<head>`, add the `<mi-map-googlemaps>` cust
 </body>
 ```
 
+> Remember to add your API keys. You can use the demo MapsIndoors API key showing "The White House": 79f8e7daff76489dace4f9f9
```
Comment (marksdk) [inline src/web/v4/getting-started/map.md:106]:
```suggestion
> Remember to add your API keys. You can use the demo MapsIndoors API key showing "The White House": `79f8e7daff76489dace4f9f9`
```

File: src/web/v4/getting-started/map.md [hunk 5]
```diff
@@ -103,6 +103,8 @@ After you added the script tag into `<head>`, add the `<mi-map-googlemaps>` cust
 </body>
 ```
 
+> Remember to add your API keys. You can use the demo MapsIndoors API key showing "The White House": `79f8e7daff76489dace4f9f9`
```
Comment (Daniel-Nielsen) [inline src/web/v4/getting-started/map.md:106]:
```suggestion
> Remember to add your API keys. You can use the demo MapsIndoors API key showing "The White House": `{{sdk.tutorialAPIKey}}`
```

**Review by Daniel-Nielsen — APPROVED** [review]

**Review by marksdk — APPROVED** [review]

File: src/web/v4/getting-started/map.md (hunk 5 above)
Comment (simonlaustsen) [reply src/web/v4/getting-started/map.md]: Variable used 👍

**Comment by simonlaustsen** [comment]
Hi @marksdk 
Can I get a final thumbs up for the latest commits?

**Review by marksdk — APPROVED** [review]

**Pull request merged**